### ISSUE TITLE
Active form is not submitted when you click submit button first time

### ISSUE THREAD

**Issue by fornit1917** [yiisoft/yii2#13531]
### What steps will reproduce the problem?

1. Create view with same code:

```
<?php
use yii\bootstrap\ActiveForm;
?>
<div class="well">
    <?php
    $form = ActiveForm::begin([
        'layout' => 'default',
        'enableClientValidation' => false,
        'enableAjaxValidation' => false,
        'options' => [
            'method' => 'POST',
            'class' => '',
            'id' => 'my-form',
            'enctype' => 'multipart/form-data',
        ]
    ]);
    ?>
        <?=$form->field($model, 'username')->textInput()?>
        <button id="submit" type="submit" class="btn btn-primary">Submit</button>
    </fieldset>
    <? ActiveForm::end(); ?>
</div>
```

$model - any instance of Model class.

2. Render this view in any action.

3. Open page with this view and click on submit button.

### What is the expected result?

Form is submitted, page reloaded, because client validation is disabled.

### What do you get instead?

Form is not submitted, page is not reloaded. Form will submitted only after second click on button.

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.11
| PHP version      | 5.6
| Operating system | Debian 8

This bug can be reproduced only in 2.0.11. In 2.0.10 it works normally.

**Comment by SilverFire** [comment]
You have problems because of submit button ID.

http://api.jquery.com/submit/

> Forms and their child elements should not use input names or ids that conflict with properties of a form, such as `submit`, `length`, or `method`. Name conflicts can cause confusing failures. For a complete list of rules and to check your markup for these problems, see [DOMLint](http://kangax.github.io/domlint/).

**Comment by SilverFire** [comment]
Related to the following jQuery bug, marked as "wontfix" as well: https://bugs.jquery.com/ticket/1414
